packages/core/lib/debug/interpreter.js

```diff
@@ -321,7 +310,7 @@ class DebugInterpreter {
 
     //quit if that's what we were given
     if (cmd === "q") {
-      return await util.promisify(this.repl.stop.bind(this.repl))();
+      process.exit();
```

Comment:
We should also eventually set the exit code here, no? Again, I don't think this should hold this PR up.